protocol-tests/src/main/java/fixtures/dpgcustomization/models/ProductReceived.java

```diff
@@ -4,23 +4,25 @@
 
 package fixtures.dpgcustomization.models;
 
+import com.azure.core.annotation.Generated;
 import com.azure.core.util.ExpandableStringEnum;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import java.util.Collection;
 
 /** Defines values for ProductReceived. */
 public final class ProductReceived extends ExpandableStringEnum<ProductReceived> {
     /** Static value raw for ProductReceived. */
-    public static final ProductReceived RAW = fromString("raw");
+    @Generated public static final ProductReceived RAW = fromString("raw");
```

Comment:
We should have the annotation in a separate line. 

```suggestion
    @Generated 
    public static final ProductReceived RAW = fromString("raw");
```

Reply:
This is due to the java formatter, we have the same issue in the first partial update pr as well: https://github.com/Azure/autorest.java/pull/1273#discussion_r764423408